conf/messages.xml

```diff
@@ -0,0 +1,2893 @@
+<?xml version="1.0"?>
+<!DOCTYPE protocol SYSTEM "messages.dtd">
```

Comment:
if you have changes to messages.xml file, make a PR for PPRZLINK and remove from this one